contracts/src/chainlink/cairo/ocr2/aggregator.cairo

```diff
@@ -565,11 +506,11 @@ func transmit{
     let (median_idx : felt, _) = unsigned_div_rem(observations_len, 2)
     let median = observations[median_idx]
 
-    # Check abs(median) is in i192 range.
-    # NOTE: (assert_lt_felt(-i192::MAX, median) doesn't work correctly so we have to use abs!)
+    # Check abs(median) is in i128 range.
```

Comment:
Let's add a test for this.

We could test the min/max values during the 'should transmit correctly' test